bin/build-test

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-image="nasa/harmony-gdal-adapter-test"
+image="ghcr.io/nasa/harmony-gdal-adapter-test"
```

Comment:
This is just internal right?

Reply:
It is, yes. It doesn't really matter, but I wanted to make this consistent with all the other services we have.